tests/test_networks.py

```diff
@@ -647,13 +647,13 @@ class Model2(BaseModel):
         Model(v='ws:///foo/bar')
 
 
-@pytest.mark.xfail(reason='URL to JSON not yet supported')
 def test_json():
     class Model(BaseModel):
         v: HttpUrl
 
     m = Model(v='http://foo@example.net')
-    assert m.model_dump_json() == '{"v": "http://foo@example.net"}'
+    # TODO: Is it okay that the trailing '/' gets added?
+    assert m.model_dump_json() == b'{"v":"http://foo@example.net/"}'
```

Comment:
Question 5: Note this behavior has changed relative to v1, is this acceptable? (I.e., adding the trailing `/`)

Reply:
Personally I think this is okay